src/mixins/IndividuMixin.js

```diff
@@ -1,8 +1,9 @@
 import Individu from '@/lib/Individu'
 
-export const createIndividuMixin = (props, required) => {
+export const createIndividuMixin = (props, optional) => {
     return {
         data: function() {
+            this.$store.dispatch('updateError', false)
```

Comment:
Pourquoi faire cet appel ici ? Ne peut(on pas déléguer cela à [la partie validation](https://github.com/betagouv/aides-jeunes/pull/1529/files#diff-83cc1ffae07b5e31a4fecdef422e25ff986de82c4205f802b4e4b598b5e23ce0R46-R50) ?

Reply:
J'imagine que c'est pour gérer les cas où les personnes font autre chose que « Suivant ». Dans le cas du bouton « Retour », je pense que le `window && window.history.back()` peut être remplacé par une vraie fonction qui fait cet appel `this.$store.dispatch('updateError', false)`.

Reply:
J'ai l'impression que ce reset de l'erreur aurait plus sa place dans les dispatch `update_*`, qu'en penses-tu ?

Reply:
Hello, oui c'est exactement pour ça, et aussi (mais c'est peut être trop edge case), le cas ou on retourne sur une page par l'url, ou le bouton back du navigateur, et dans ce cas la fonction de retour contenant `window && window.history.back()` et `this.$store.dispatch('updateError', false)` n'est pas appelée.

Reply:
Ça pourrait plutôt être dans [`mounted`](https://vuejs.org/v2/api/#mounted) dans ce cas là.

Reply:
Celui-ci est en trop d'ailleurs, c'est celui dans action que souhaitais garder. (Et je sais que ce n'est pas très propre mais c'était la façon de gérer les pbs dont on parle dans les commentaires ci-dessus)

Reply:
Oui dans `mounted`, en effet, j'ai pas récupéré encore tout mes automatismes sur vue. Par contre dans le composant `Actions` si on veut que ce soit partout sans trop changer de choses en attendant d'avoir les updates design de Clément. Sinon IndividuMixin + tous les autres écrans (menages, familles, ...)

> J'ai l'impression que ce reset de l'erreur aurait plus sa place dans les dispatch update_*, qu'en penses-tu ?

Je suis désolé, je ne vois pas ce que tu veux dire : /